activestorage/lib/active_storage/engine.rb

```diff
@@ -102,7 +101,8 @@ class Engine < Rails::Engine # :nodoc:
               ]
             end
 
-          ActiveStorage.analyzers = [analyzer].compact.concat(app.config.active_storage.analyzers || [])
+          ActiveStorage.analyzers = app.config.active_storage.analyzers ||
```

Comment:
I think this behaviour make the most sense and it address the concern from https://github.com/rails/rails/pull/54402#issuecomment-2623876462.

But I just want to point that we now have the reverse problem and it introduces a behaviour change (where one could set its own analyzer and also have the default one).

Do you think we should for now just deprecated the old behaviour and in a future rails release introduce this change?